recipe/patches/0001-Remove-PySpark3-kernel.patch

```diff
@@ -0,0 +1,506 @@
+diff --git a/Dockerfile.jupyter b/Dockerfile.jupyter
```

Comment:
This should just be right next to `meta.yaml`. No need to put it in a `patches` directory.